phpphp/homework3.php

```diff
@@ -0,0 +1,19 @@
+<?php
+//homework3
+function checkPrimeNumber ($n) {
+  if ($n === 0 || $n === 1)  {
+    return false;
+  }
+  if ($n === 2) {
+    return true;
+  } 
+  $m = $n/2;
+  for($i = 2; $i <= $m; ++$i) {
+      if ($n%$i == 0) {
+        return false;
+        break;  
```

Comment:
`return false;` will exist function... `checkPrimeNumber()`, is it right? so do we need `break;` ?

Reply:
don't need `break` because `return` for itself can finish function. So, I'll delete it.